client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/Layout/Layout.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useMemo } from 'react';
+import {
+  Grid,
+  SxProps,
+  Theme,
+  Typography,
+  useFormatNumber,
+  useIntlUtils,
+  useTranslation,
+} from '@openmsupply-client/common';
+import {
+  getValueInUnitsOrPacks,
+  Representation,
+  RepresentationValue,
+} from '../utils';
+
+interface LayoutProps {
+  Top: React.ReactElement;
+  Left: React.ReactElement | null;
+  Middle: React.ReactElement | null;
+  Right: React.ReactElement | null;
+}
+
+export const Layout = ({ Top, Left, Middle, Right }: LayoutProps) => {
```

Comment:
Whole file can be re-used for Requisitions as well... will do some shufflin' when I start the Requisition refactor